Build/BuildPackage.ps1

```diff
@@ -25,9 +25,11 @@ $logFile = Join-Path $PSScriptRoot BuildPackage.log
 
 Write-Host "Exporting Xbox Live Unity Plugin to " -NoNewline
 Write-Host $packagePath -ForegroundColor Green
+Write-Host "$($unity) -batchmode -logFile '$($logFile)' -projectPath '$($projectPath)' -exportPackage '$($exportAssetPath)' '$($packagePath)' -quit"
```

Comment:
@jasonsandlin You don't need the extra $() around those variables.  You only need that when you need to evaluate something like $($unity.Length)